bindings/python/mata.pyx

```diff
@@ -501,20 +773,28 @@ cdef class Nfa:
 
     @classmethod
     def remove_epsilon(cls, Nfa lhs, Symbol epsilon):
-        """Removes transitions that contains epsilon symbol
+        """Removes transitions that contain epsilon symbol
 
         TODO: Possibly there may be issue with setting the size of the automaton beforehand?
 
         :param Nfa lhs: automaton, where epsilon transitions will be removed
         :param Symbol epsilon: symbol representing the epsilon
         :return: automaton, with epsilon transitions removed
         """
-        result = Nfa(lhs.state_size())
-        mata.remove_epsilon(
-            result.thisptr, dereference(lhs.thisptr), epsilon
-        )
+        result = Nfa()
```

Comment:
I guess, removal of epsilon was fixed, so it does not need to "guess" the size of the automaton beforehand. Am I right?

Reply:
I am not sure if I understand what do you mean by “guess”. Could you elaborate?

There is a function `Mata::Nfa::remove_epsilon()` in Mata which is currently used. I have only updated the previous function by increasing the size of the result automaton when the automaton is not large enough by itself.

Reply:
Great. I was not sure if this was what you were referring to. I consider this conversation resolved.